src/js/pyodide.ts

```diff
@@ -168,6 +169,11 @@ export async function loadPyodide(
      * Make loop.run_until_complete() function correctly using stack switching
      */
     enableRunUntilComplete?: boolean;
+    /**
+     * If false (default), throw an error if the version of Pyodide core does not
```

Comment:
```suggestion
     * If true (default), throw an error if the version of Pyodide core does not
```

and change the default value of the comment